### ISSUE TITLE
 math didn't go according to plan

### ISSUE THREAD

**Issue by RangeBanenge** [Ajneb97/ConditionalEvents#62]
https://github.com/Ajneb97/ConditionalEvents/assets/123173065/8e0ee81e-c31e-4d00-b78c-877ce0591764

**Comment by RangeBanenge** [comment]
bedwars_giveitems:
    type: item_pickup
    allow_math_formulas_in_conditions: true
    enabled: true
    conditions:
    - '%item% == COPPER_INGOT'
    - '(%math_0:_{checkitem_amount_mat:COPPER_INGOT}+{item_amount}% % 64) == 0 execute giveNaturally'
    - '(%checkitem_amount_mat:COPPER_INGOT% % 64) == 0'
    actions:
      giveNaturally:
        - 'give_item: id:%item%;amount:%item_amount%;slot_replace:false;name:&4Медь'
        - 'wait_ticks: 1'
        - 'remove_item: %item%;%item_amount%'
      default:
        - 'give_item: id:%item%;amount:%item_amount%;slot:%math_0:_{player_empty_slots}-1%;slot_replace:false;name:&4Медь'
        - 'wait_ticks: 1'
        - 'remove_item: %item%;%item_amount%'
  bedwars1:
    type: item_pickup
    enabled: true
    conditions:
    - '%item% == COPPER_INGOT'
    actions:
      default:
      - 'actionbar: &7┃ &r&4%checkitem_amount_mat:COPPER_INGOT%&r &7┃ &r&7%checkitem_amount_mat:IRON_INGOT%&r &7┃ &r&6%checkitem_amount_mat:GOLD_INGOT%&r &7┃ &r&0%checkitem_amount_mat:NETHERITE_INGOT%&r &7┃;0'

**Comment by Ajneb97** [comment]
I don't think you are using the variables correctly, why are you adding parenthesis before the variables?
